Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -335,20 +322,20 @@ static void CheckCPConfiguration(int vtx_attr_group)
   }
 }
 
-int RunVertices(int vtx_attr_group, OpcodeDecoder::Primitive primitive, int count, DataReader src,
-                bool is_preprocess)
+template <bool IsPreprocess>
+int RunVertices(int vtx_attr_group, OpcodeDecoder::Primitive primitive, int count, DataReader src)
 {
   if (count == 0)
     return 0;
   ASSERT(count > 0);
 
-  VertexLoaderBase* loader = RefreshLoader(vtx_attr_group, is_preprocess);
+  VertexLoaderBase* loader = RefreshLoader<IsPreprocess>(vtx_attr_group);
 
   int size = count * loader->m_vertex_size;
   if ((int)src.size() < size)
     return -1;
 
-  if (is_preprocess)
+  if (IsPreprocess)
```

Comment:
`if constexpr (IsPreprocess)` ?

Reply:
Just curious, was there not an advantage here?

Reply:
Oops, I actually made that change but lost it in a rebase.

Reply:
It looks like using `if constexpr (IsPreprocess)` angers MSVC... but only in debug builds? I'm checking if there's any workaround.

Reply:
https://developercommunity.visualstudio.com/t/VS2019-16713-Incorrect-C4702-unreachab/1440640 is the issue. The only workaround I can see is to surround the entire rest of the code with `if constexpr (!IsPreprocess) { ... } return size;` instead.